tests/chainer_tests/links_tests/connection_tests/test_convolution_nd.py

```diff
@@ -125,6 +125,18 @@ def test_pickling(self, backend_config):
 
         testing.assert_allclose(y_data1, y_data2, atol=0, rtol=0)
 
+    def test_from_params(self, backend_config):
+        link1 = self.create_link(self.generate_params())
+        link1.to_device(backend_config.device)
+
```

Comment:
Add here this, otherwise parameters will be unitialized and the test will fail
```python
if self.in_channels in (None, 'omit'):
            link1._initialize_params(self.x_shape[1])
        link2 = convolution_nd.ConvolutionND.from_params(
            link1.W, link1.b,
            stride=self.stride, pad=self.pad, groups=self.groups)
```

Reply:
How about skipping tests if `self.in_channels` is `None` or `'omit'`?

Reply:
IMO, if `self.in_channels` is `None` or `'omit'`, it'd be ok to skip this test.

What do you think?

Reply:
IMO, if `self.in_channels` is `None` or `'omit'`, it'd be ok to skip this test.

What do you think?

Reply:
Its an option too.